helm-cluster-scoped/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-name: ibm-iam-operator-cluster-scoped-resources
+name: ibm-iam-operator-cluster-scoped
 description: Cluster-scoped Resources for IBM IAM Operator
 type: application
-version: "4.11.0"
+version: "4.11.0+20250225.133605.1"
```

Comment:
Please remove the build metadata.  This will be added by the build pipeline and should not be included in the operator repo.